madgiraffestudios/src/components/About/About.scss

```diff
@@ -2,17 +2,39 @@
 
 .description-text { width: 75%; }
 
+#photoshoots{ margin-top: 258px;}
+
+#graphic-design{margin-top: 40px;}
+#second{ margin-bottom: 100px;}
+#illustrations{ margin-top: 415px;}
+.about-img{
+    box-shadow: 5px 10px 18px #888888;
+ }
+
+ .bottom-img{
+     margin-bottom: 100px;
+ }
 @media only screen and (max-width:768px) {
-    .studio-text { font-size: 20px; text-align: center; width: auto; }
-   .studio-name{text-align: center; font-size: 30px;  }
+    .studio-text { font-size: 17px; text-align: center; width: 99%; }
+   .studio-name{ font-size: 1.5rem;  }
     
-   .description-text { font-size: 30px; text-align: center; width: auto; }
-   .description-title{text-align: center;}
+   .description-text { font-size: 20px; text-align: center; width: 99%; }
+   .description-title{text-align: center; font-size: 30px; }
 
    .about-img{ 
     box-shadow: 5px 10px 18px #888888;
     margin-bottom: 60px;
     }
+
+    #photoshoots{ margin-top:0px;}
```

Comment:
all these ids can be referenced in one line I believe, let's have a look today. This should be 0 instead of 0px